### ISSUE TITLE
How can I use CPU only?

### ISSUE THREAD

**Issue by mintlatte** [weiliu89/caffe#479]
I want to use only CPU to execute ssd_pascal_video.py etc.
So, I tried to change solver.protxt & ssd_pascal_video.py

Like this ------
solver.mode: CPU

solver_mode = P.Solver.CPU
-----------

But detection windows was frozen, so I just quitted it.

Is any other options I have to change?

Please tell me about that.

Thank you.

**Comment by runauto** [comment]
just same with me(1.0s per frame ), I use only_cpu mode, my cpu E5-1603,2.8GHz, I don't know why.

**Comment by zlbing** [comment]
so anyone figure out it ?

**Comment by jinxuan777** [comment]
@mintlatte could you get correct result in CPU_only mode

**Comment by hengchuan** [comment]
ooo

**Comment by ujsyehao** [comment]
@jinxuan777 Yes, I can get the correct result in CPU mode, the fps is 0.35 on GTX 1070 ti
